ALUM/ALUM/Views/SignUpPage/SignUpSetUp.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SignUpSetUp.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 2/27/23.
+//
+
+import SwiftUI
+
+struct SignUpSetUp: View {
```

Comment:
Let's make it a practice to have `Screen` as suffix for all Views that represent whole screens.

These naming conventions will make our codebase more readable.